grafana-plugin/src/containers/OutgoingWebhook2Form/OutgoingWebhook2Form.tsx

```diff
@@ -52,21 +52,22 @@ const OutgoingWebhook2Form = observer((props: OutgoingWebhook2FormProps) => {
   return (
     <Drawer
       scrollableContent
-      title={
-        <Text.Title className={cx('title')} level={4}>
-          {action === 'new' ? 'Create' : 'Edit'} Outgoing Webhook
-        </Text.Title>
-      }
+      title={action === 'new' ? 'Create Outgoing Webhook' : 'Edit Outgoing Webhook'}
       onClose={onHide}
-      closeOnMaskClick
+      closeOnMaskClick={false}
     >
       <div className={cx('content')} data-testid="test__outgoingWebhook2EditForm">
         <GForm form={form} data={data} onSubmit={handleSubmit} />
-        <WithPermissionControlTooltip userAction={UserActions.OutgoingWebhooksWrite}>
-          <Button form={form.name} type="submit" disabled={data.is_legacy}>
-            {action === 'new' ? 'Create' : 'Update'} Webhook
+        <HorizontalGroup>
```

Comment:
Is this right-aligned in the drawer? haven't pulled the branch but at the first glance this might need `justify-content={'flex-end'}`

Reply:
Good point, will update :+1: